lib/SpeculationModules/SlampOracleAA.cpp

```diff
@@ -36,7 +69,9 @@ LoopAA::ModRefResult SlampOracleAA::modref(
 {
   //std::string space(getDepth()+1, ' ');
   //errs() << space << "si\n";
-  return LoopAA::modref(A,rel,ptrB,sizeB,L,R);
+  //YEBIN: hack
```

Comment:
@yebinchon 
We do not need this hack anymore. Same applies to other ones in this file.
Check the memdep API. It's the high-level API used by remediator. It will not ask for other LoopAA. The workflow now is after getting the PDG, ask each remediator to see any dependences can be removed with the `satisfy` or `memdep` API.